assets/src/apps/v2/on_bus.tsx

```diff
@@ -8,31 +8,68 @@ import "../../../css/on_bus_v2.scss";
 
 import React from "react";
 import ReactDOM from "react-dom";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+} from "react-router-dom";
```

Comment:
Formatter weirdness? This seems to be exactly the same statement before and after.